windows_2008_r2.json

```diff
@@ -5,7 +5,7 @@
       "iso_url": "{{user `iso_url`}}",
       "iso_checksum_type": "{{user `iso_checksum_type`}}",
       "iso_checksum": "{{user `iso_checksum`}}",
-      "headless": true,
+      "headless": false,
       "boot_wait": "2m",
       "ssh_username": "vagrant",
```

Comment:
When having a look at programs that can be uninstalled. I noticed OpenSSH was one of the options. In here it looks like we actually connect over SSH. Otherwise you would have to specify the communicator of winrm (Ex. https://git.dev.d2l/users/mtjandrawidjaja/repos/thebaker/browse/lms_baker.json?at=refs%2Fheads%2FRedekop#34,42-44). On my first smoke test of your image, I was actually able to connect over SSH, resulting in packer trying to upload files with `scp` which ended up failing for me.

- [ ] Use WinRM and do not install OpenSSH server if it is not needed/wanted